src/main/kotlin/persistence/DirectoryProvider.kt

```diff
@@ -47,4 +48,21 @@ class DirectoryProvider(private val appName: String) : IDirectoryProvider {
         file.mkdirs()
         return file
     }
+
+    // Creates a user subdirectory and stores their hash info
+    fun getAudioDirectory(): File {
+        return createInternalAppDirectory("Audio")
+    }
+
+    // Stores the image with name of the user who stored it, followed by the original name of the file
+    fun storeImageDirectory(userMine: User, imageFile: File): File {
+        return createInternalAppDirectory("Images")
```

Comment:
Same as audio function, change to something like `getAppDataDirectory("users${separator}images")`